platform/lib/domain/src/main/kotlin/io/hamal/lib/domain/_enum/ExecLogLevel.kt

```diff
@@ -1,9 +1,18 @@
 package io.hamal.lib.domain._enum
 
-enum class ExecLogLevel {
-    Trace,
-    Debug,
-    Info,
-    Warn,
-    Error
+enum class ExecLogLevel(val value: Int) {
+    Trace(0),
```

Comment:
I suppose this written to the database - lets not start with 0 - instead with 1 - the reason is: Sometimes things default to 0 which causes unexpected behavior --> I rather have an exception